tests/installation/bootloader_zkvm.pm

```diff
@@ -92,5 +92,21 @@ sub run {
         }
     }
 }
+sub post_fail_hook {
+    reset_consoles;
+    select_console 'svirt';
+
+    # Enter Linuxrc extra mode
+    type_line_svirt 'x';
+    wait_serial "Linuxrc extras";
+
+    # Start linuxrc shell
+    type_line_svirt '3';
+    wait_serial "ttysclp0:install";
+
+    # Collect Linuxrc logs
+    type_line_svirt "'cat /var/log/linuxrc.log > /dev/$serialdev && echo 'LINUXRC_LOG_SAVED' > /dev/$serialdev'";
+    wait_serial "LINUXRC_LOG_SAVED" ? record_info 'Logs collected', 'Linuxrc logs can be found in serial0.txt' : die "could not collect linuxrc logs";
```

Comment:
For all three type_line_svirt+wait_serial tuples: `type_line_svirt` has the "expect" parameter, see opensusebasetest::rewrite_static_svirt_network_configuration how it's used. So the section could/should be rewritten as:

```perl
    # Enter Linuxrc extra mode
    type_line_svirt 'x', expect => 'Linuxrc extras';

    # Start linuxrc shell
    type_line_svirt '3', expect => 'ttysclp0:install';

    # Collect Linuxrc logs
    type_line_svirt "'cat /var/log/linuxrc.log > /dev/$serialdev && echo 'LINUXRC_LOG_SAVED' > /dev/$serialdev'";
￼    wait_serial "LINUXRC_LOG_SAVED" ? record_info 'Logs collected', 'Linuxrc logs can be found in serial0.txt' : die "could not collect linuxrc logs";
```

Reply:
Nice. Didn't know that. Will change